packages/wonder-stuff-sentry/src/types.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import type {Metadata} from "@khanacademy/wonder-stuff-core";
+
+/**
+ * Tags for a Sentry event.
+ *
+ * The key of each item is the name of the tag.
+ */
+export type SentryTags = {
```

Comment:
Even though the Sentry API allows for arbitrary tags, I locked this down to a set list of known-tags for a few reasons:
- Adding tags means they're always there in our Sentry project, and I wanted to avoid "Tag Proliferation"
- Avoids misspellings (which leads to "Tag Proliferation" and confusion on which is the right tag to look at when in the Sentry dashboard trying to find issues and stuff)
- Makes the types for each Tag clear to the developer
- Allows you to search Tags _across_ issues